parts/agentparams.t

```diff
@@ -92,7 +92,7 @@
       "type": "string"
     },
     "{{.Name}}osImageVersion": {
-      "defaultValue": "16.04.201804050",
+      "defaultValue": "latest",
```

Comment:
not changing the actual default value, just the default that appears in the parameters json to reflect reality